src/Component/ActionContainer.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+function ActionContainer(props) {
+  const handleAttack = () => {
+    const randomNumber = Math.floor(Math.random() * (10)) + 1
```

Comment:
this logic is repeating and can be abstracted in a helper file

Reply:
Done